test-it/karma/javascriptinvoke/DataTypes.Test.js

```diff
@@ -108,6 +116,14 @@ describe('A JavaScript function invoke', function () {
             expect(window.NI_SingleFunction).toHaveBeenCalledWith(3.0);
             expect(window.NI_DoubleFunction).toHaveBeenCalledWith(6.0);
             expect(window.NI_StringFunction).toHaveBeenCalledWith('National Instruments');
+            expect(window.NI_ArrayFunction).toHaveBeenCalledWith(Int8Array.from([1, 2, 3]));
```

Comment:
For integer types, please add the Min and Max value in the integer range. For floating point values please do negative, zero, positive values, at this moment I don't have a better idea about testing them